src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -1469,7 +1469,7 @@ FullWidthRepeat:
         End Function
 
         Private Function TokenOfStringCached(spelling As String) As SyntaxKind
-            If spelling.Length = 1 OrElse spelling.Length > 16 Then
+            If spelling.Length > 16 Then
```

Comment:
For what it's worth, this was my change (back when I prototyped this) and I reviewed it with @VSadov at the time.  We concluded it was safe, but I can't recall all the details.